src/lib/inputs/InputRadio/InputRadioGroup.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC } from 'react';
+import classNames from 'classnames';
+import { FormikProps } from 'formik';
+import { FormControl, FormLabel, RadioGroup, RadioGroupProps } from '@material-ui/core';
+
+export interface InputRadioGroupProps extends RadioGroupProps {
+  className?: string;
+  formikProps?: FormikProps<any>;
+  legend?: string;
+}
+
+export const InputRadioGroup: FC<InputRadioGroupProps> = ({
+  children,
+  className,
+  formikProps,
+  legend,
+  name,
+  onChange,
+  row,
+  value
+}) => {
+  const fieldProps = formikProps?.getFieldProps(name);
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    if (onChange) onChange(event, event.target.value);
+    if (fieldProps?.onChange) fieldProps.onChange(event);
+  };
+
+  return (
+    <FormControl className={classNames('lc-input-radio-group', className)} component="fieldset">
+      <FormLabel component="legend">{legend}</FormLabel>
+      <RadioGroup aria-label={name} name={name} onChange={handleChange} row={row} value={value}>
+        {children}
+      </RadioGroup>
```

Comment:
above if you had the props destructured like this
`({className, formikProps, legend, onChange, ...props})`

then you can have
`<RadioGroup aria-label={props.name} onChange={handleChange} {...props} />`
because children row value name would all be included in props.

I'm trying to encourage less destructuring/restructuring

Reply:
wait just noticing that value can be assumed if there is formikProps

`({className, formikProps, legend, onChange, value, ...props})`

```
const value = value ?? fieldProps?.value;

<RadioGroup aria-label={props.name} onChange={handleChange} value={value} {...props} />
```

Reply:
That makes sense! This is resolved.